w_common/lib/src/common/disposable.dart

```diff
@@ -514,8 +514,9 @@ class Disposable implements _Disposable, DisposableManagerV7, LeakFlagger {
 
   @mustCallSuper
   @override
-  T manageAndReturnTypedDisposable<T extends Disposable>(T disposable) {
-    _throwOnInvalidCall('manageAndReturnDisposable', 'disposable', disposable);
+  T manageAndReturnTypedDisposable<T extends Disposable?>(T disposable) {
```

Comment:
**Suggestion**: Here too I think it might read better this way (though it just might be the way my brain works):

```suggestion
  T? manageAndReturnTypedDisposable<T extends Disposable>(T? disposable) {
```